dependencies/url/validator.go

```diff
@@ -59,7 +72,7 @@ func init() {
 	}
 }
 
-//  isPrivateIP reports whether an IP exists in a known private IP space.
+//  IsPrivateIP reports whether an IP exists in a known private IP space.
```

Comment:
Did you mean to change this?

Reply:
Oops, I made it public in an intermediate version, but then changed it back after adding `ValidateIP` to the `Validate` Interface. Then forgot to fix the comment. Will fix.